src/components/accordion-item/accordion-item.tsx

```diff
@@ -272,6 +271,7 @@ export class AccordionItem implements ConditionalSlotComponent {
   //--------------------------------------------------------------------------
 
   private determineActiveItem(): void {
+    this.selectionMode = getElementProp(this.el, "selection-mode", "multiple");
```

Comment:
Instead of checking here, could this be set by the parent Accordion when selectionMode is updated `@Watch("selectionMode") ...` ?

Reply:
i don't think that can be used here considering we are determining the selection of items inside the accordion-item.